stdlib/src/locale.rs

```diff
@@ -15,17 +23,64 @@ mod _locale {
         ptr,
     };
 
+    #[cfg(windows)]
+    #[repr(C)]
+    struct lconv {
+        decimal_point: *mut libc::c_char,
+        thousands_sep: *mut libc::c_char,
+        grouping: *mut libc::c_char,
+        int_curr_symbol: *mut libc::c_char,
+        currency_symbol: *mut libc::c_char,
+        mon_decimal_point: *mut libc::c_char,
+        mon_thousands_sep: *mut libc::c_char,
+        mon_grouping: *mut libc::c_char,
+        positive_sign: *mut libc::c_char,
+        negative_sign: *mut libc::c_char,
+        int_frac_digits: libc::c_char,
+        frac_digits: libc::c_char,
+        p_cs_precedes: libc::c_char,
+        p_sep_by_space: libc::c_char,
+        n_cs_precedes: libc::c_char,
+        n_sep_by_space: libc::c_char,
+        p_sign_posn: libc::c_char,
+        n_sign_posn: libc::c_char,
+        int_p_cs_precedes: libc::c_char,
+        int_n_cs_precedes: libc::c_char,
+        int_p_sep_by_space: libc::c_char,
+        int_n_sep_by_space: libc::c_char,
+        int_p_sign_posn: libc::c_char,
+        int_n_sign_posn: libc::c_char,
+    }
+
+    #[cfg(windows)]
+    extern "C" {
+        fn localeconv() -> *mut lconv;
+    }
```

Comment:
i'd like to put this function out of this module. Then it can be called like
```rust
unsafe { super::localeconv() }
```
below

Reply:
so you mean this function is in a `common` module so that it can be used in other module?